.github/workflows/main.yml

```diff
@@ -0,0 +1,197 @@
+name: Build-Deploy
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+jobs:
+  bygg-app:
+    name: Bygg app/image, push til github
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+      - name: Define build environment
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo ::set-env name=IMAGE_REGISTRY::docker.pkg.github.com/$(echo ${GITHUB_REPOSITORY})
+          echo ::set-env name=IMAGE_NAME::$(echo ${GITHUB_REPOSITORY##*/})
```

Comment:
her setter vi IMAGE_NAME for environment til soknadsfillager
se linje 41, der settes den samme variableen som .docker_image
samt
kunne Linje 39 vært satt med samme?